packages/delivery-service/src/persistence/pending/index.ts

```diff
@@ -0,0 +1,5 @@
+import { addPending } from './addPending';
```

Comment:
I think the entire storage dir should no longer be part of ds since it has moved to BE

Reply:
yes: b97020fb0befefee621e00c0547d7f401f6f6864